packages/crumbs-vue/components/VeeGrid/VeeRow/VeeRow.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+  <div :class="veeClass" :style="veeStyle">
+    <slot></slot>
+  </div>
+</template>
+
+<script lang="ts">
+import { PropType, defineComponent } from 'vue';
+import composables from '../../../composables';
+import { classBuilder } from 'crumbs-core/builders/class';
+import { VeeRowAlign, VeeRowAlignContent, VeeRowJustify } from './vee-row';
+
+export default defineComponent({
+  name: 'VeeRow',
+
+  props: {
+    dense: {
+      type: Boolean,
+      default: false,
+    },
+
+    noGutters: {
+      type: Boolean,
+      default: false,
+    },
+
+    align: {
+      type: String as PropType<VeeRowAlign>,
+      default: null,
+      validator: (value: string): boolean => {
+        return Object.values(VeeRowAlign).includes(value as VeeRowAlign);
+      },
```

Comment:
This validator is repeated all along these props. Just place it in a method for reusability and maintainability.